### ISSUE TITLE
Anamorphic pictures not displayed correctly

### ISSUE THREAD

**Issue by jclement46** [gkv311/sview#71]
Thank you for the JPS option.  2X horizontally compressed videos are displayed ok, but anamorphic pictures are not properly displayed.  I have put some samples in:
https://1drv.ms/u/s!AgEkZSUlkAcx3H4y-R4pWn52fpn8?e=25eJA5

The -LRQ works for videos, but not for pictures.  TriDef does obey the Q descriptor for pictures.  I don't know how many people have such pictures, but for completeness this should be fixed.  There is no rush for this.

**Comment by jclement46** [comment]
This problem happened on the Windows version, but not on the Android version.

**Comment by jclement46** [comment]
Using my test pictures I also found that MPO is not displaying 3D on either of my Android phones with very different version of Android.  MPO does display properly on My Windows machine.

**Comment by jclement46** [comment]
It seems that in Windows, sometimes sView does display anamorphic correctly, but I have not been able to figure out when this happens.  Mostly it displays them incorrectly,

**Comment by gkv311** [comment]
I do understand some reasons for anamorphic videos to exist, but anamorphic images is somewhat weird - they may appear only as direct snapshots taken from anamorphic video. I don't know any image editor which considers images to provide non-square pixels.

**Comment by jclement46** [comment]
Yes, they are weird, but some do exist.  One reason for anamorphic pictures is that you can add a 3D attachment to a camera and get only portrait 3D, but if it has an anamorphic lens, you can get landscape images.  There are apparently legacy anamorphic pictures.  In either case the developing windows version does not obey the suffixes to decompress anamorphic images.  But as I pointed out I got it to work once, but I do not know what incantations I used, and after exiting, snd restrating sView it still had the problem.  I just have sample anamorphic images, but loads of anamorphic videos.

Perhaps sView should have a guess option just for videos which also have the 3D tag set.  So a 720x480 or 640x480 pixel count aspect ratio could be interpreted as a 4x3 SBS, while a 16:9 could be interpretd as a 16x9 SBS.  A 720x480 pixel ratio with a 4:3 tag in non 3D tagged video should be displayed as 4x3, but if a 16:9 tag exists then it is a 16:9 video.  I think PAL does not have quite the same problems as the NTSC format.  If it is physically 720x480 and is tagged as 3D you know it is probably SBS anamorphic  so the choice is between 16:9 or 4:3, either of which would look better than the squashed raw image.  A 1440x480 would obviously be a 4:3 non-anamorphic.  I think the display aspect ratio tag should be the guide to display format, and it is correct for commercial non 3D NTSC dvds, and field sequential DVDs.  But in the future I will try to put the correct suffixes on my videos.  TriDef tries to guess whether a video or picture is anamorphic, and is OK for non 3D tagged videos, but messes up bigtime for pictures.  3D SBS scans of Realist format slides, or View-Master images are displayed wrong in TriDef.  I agree that the default for pictures should absolutely be non-anamorphic when just the 3D tag or JPS is present.

**Comment by jclement46** [comment]
As an update on MPO, it fails to display under a specific condition.  If a Fuji AVI is display, and then an MPO it fails to recognize it as MPO.   This happens on both the Windows and Android versions.  I have put the sample AVI & MPO in a box at:
https://1drv.ms/u/s!AgEkZSUlkAcx3H4y-R4pWn52fpn8?e=z4T8Ew

The ability to intermix pictures and videos works fairly well except for this particular glitch.  Of course the anamorphic tagged pictures do not display correctly in Windows, as it has not been updated recently, but they are OK in Android.
